.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,82 @@
+name: Report an issue
+description: Create a bug report to fix an issue
+title: 'Issue: '
+labels: 'bug'
+body:
+  - type: markdown
+    attributes:
+      value: >
+        :warning: **Please do not report security vulnerabilities here**
+
+        Be careful with sensitive information and security vulnerabilities. In order to report bugs that could contain
+        sensitive information, use [Launchpad](https://bugs.launchpad.net/ubuntu/+source/adsys/+filebug) instead.
+        On Ubuntu machines, you can use `ubuntu-bug adsys` to collect relevant information.
+
+
+        Thanks for taking the time to report an issue and help improve adsys! Please fill out the form below as
+        best as you can so that we can help you.
+
+        Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general
+        support or usage questions, use [Ubuntu Discourse](https://discourse.ubuntu.com/c/desktop/8).
+
+        By submitting an Issue to this repository, you agree to the terms within the
+        [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct).
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      options:
+        - label: I have searched the existing issues and found none that matched mine
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the issue
+      description: >
+        Provide a clear and concise description of what the issue is, including what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce it
+      description: >
+        Detail the steps taken to reproduce this error, what was expected, and whether this issue can be reproduced
+        consistently or if it is intermittent.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+
+        Please include screenshots (you can drag any image here) where applicable. Redact any sensitive information.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Ubuntu users: System information"
+      description: >
+        Ubuntu users can run `ubuntu-bug adsys --save=/tmp/report.txt` and drag the file below.
+        It will contain any useful information.
+  - type: textarea
+    attributes:
+      label: "Non Ubuntu users: System information"
+      description: |
+        For users of other distributions than ubuntu, provide details about the environment you experienced the issue in:
+      value: |
+        ### Environment
+        * adsys version: please run `adsysctl version`
+        * Distribution: (**NAME** in `/etc/os-release`)
+        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
```

Comment:
Shouldn't that be in a separate textarea? Having all globed together can get a bit confusing and annoying to understand what's happening.

Reply:
I don’t think so, this is really "debug info" with environment + logs.

This is not needed on ubuntu machine, because the apport hook is supposed to collect the logs too, and will be in the previous paragaph.

Reply:
But doesn't it kind of defeat the whole purpose, then? It would be still 70% of the info we need inside a markdown textarea that users can choose to ignore and then we go back to square 1, no?

Reply:
Not for the logs themselves. Most of the time, people will use ubuntu and file up the other field.

Reply:
It would be a completely different case if GitHub actually rendered the markdown that we add as the value for this field, but it's just a lot of text with ``` and ## that is a bit annoying to navigate through... 
 Also, most of the users don't use apport to provide us with the report, I'm not sure why. Maybe it's a bit "scary"?

Reply:
Right, but you can say:
field 1: "this is for ubuntu users"
field 2: "for non ubuntu user, please set the environment here"
field 3: "for non ubuntu user, please file the logs here"

I think it needs to have some symetry.

Reply:
I agree. So let's rename the relevant information field to something in the way of "logs for non Ubuntu users".
Also, I think we should add a note that it's for non-Ubuntu and for users that use the package through a PPA as well, since apport does not work on sources outside of the archive. 
Wdyt?

Reply:
Well, the section title is relevant "Non Ubuntu users: System information", which they are: logs, environment. Things that are coming from the system.

If you want, we can add a `### Logs` section to it.

On apport, as you sure if you do the report to a file, this is prevented too? Do you mind giving a test if there is a switch for it?

Reply:
Oh, the `System Information` section is fine for me... I was referring to the following `textarea` labeled as `Relevant Information`... Maybe we can rename this to something like: `Additional Information (logs, some other reports, etc.)`

Reply:
I don’t think the logs should be in relevant information. It should be in the current textarea.

Reply:
Let's hope the users follow that, then. I wanted something more explicit so that we can incentivize them to submit it the logs with the report, but if you feel this is enough, we can try it.

Reply:
Also, about `ubuntu-bug`, if you try to use it on a package that is on a PPA, you get: `This does not seem to be an official Ubuntu package`, and then it asks me to update the indexes, remove third party packages and try again.